docs/installation.rst

```diff
@@ -0,0 +1,27 @@
+.. _sec_installation:
+
+============
+Installation
+============
+
+Requirements
+------------
+
+``Demes`` uses `srictyaml <https://github.com/crdoconnor/strictyaml>`_ to 
+read YAML-defined demographic models.
+
+Dependencies should be installed automatically when installing ``demes``
+via pip or conda.
+
+Conda
+-----
+
+.. note::
+   To be made availabe through bioconda (or similar) upon release.
+
+Pip
+---
+
+.. note::
+   Fill in when ready.
+
```

Comment:
Maybe best not get too far ahead of ourselves here. `demes` is definitely not ready for users to install and use. It might be better to be explicit about that, and suggest that in its current state, `demes` is useful only for developers.

Installation instructions for developers might look something like:
```
## Clone the repository
$ git clone https://github.com/grahamgower/demes.git
$ cd demes

## Install the developer dependencies
$ python -m pip install -r requirements.txt

## Run the test suite
$ python -m pytest -v tests
```